### DISCUSSION TITLE
关于『禁漫天堂』网站图标问题

### DISCUSSION THREAD

**Discussion by taitekisan** [F9y4ng/GreasyFork-Scripts#285] [问答专区(Question / Answer)] [answered]
浏览器：Google Chrome 119.0.6045.124
脚本管理器：Tampermonkey v4.19.0
脚本版本：字体渲染（自用脚本） 2023.11.07.1
问题网站：18comic.vip
![捕获](https://github.com/F9y4ng/GreasyFork-Scripts/assets/97374908/57af33a8-ad56-4ed0-a8b8-a28df005cc71)

**Comment by F9y4ng** [comment, marked as answer]
我这里显示是正常的。
### 建议
旧版升级至新版的用户注意：在遇到样式问题时，请先尝试使用重置按钮重置数据，再重新配置相关参数，以使用最新过滤规则。

**Reply by taitekisan** [reply]
你好，
在重新安装了插件以及重启浏览器该问题已得到解决。
谢谢你的回复。
